lte/gateway/python/magma/subscriberdb/tests/client_tests.py

```diff
@@ -288,6 +300,14 @@ def get_all_subscribers(self):
             ),
         ]
 
+    @ unittest.mock.patch(
```

Comment:
this should not be separated by whitespace

Reply:
Hi @ardzoht ,
updated the format as suggested.